CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+
+---
+
+## Dev workflow
+
+Quick overview of the workflow:
+
+1. bug is found / feature is suggested → new Issue
+2. new branch is created with code solving the Issue → Push Request initiated
+3. after PR is finished, tested and approved (by at least 1 Team member) → merge to `dev` branch
+4. when multiple PRs are merged, new Release is prepared → merged to `master` branch
+
+---
+
+### 1. Issues
+
+When bug is found or new feature/suggestion proposed, [new Issue gets created](https://github.com/particl/partgui/issues). Our repo has predefined Issue template to help you started with that (the more info you provide the better).
+
+- **Always keep the Issue's title short and descriptive!**
+  - Examples:
+    - ✅ _"Pagination in Address book doesn't work (macOS)"_
+    - ❎ _"When you click next page in address book nothing happens and 2 pages are hidden please help!!!1!"_
+- **Include as much related information** in Issue's description as you can (the predefined template will help you with that)
+  - For bugs: what's happening, how to reproduce it, what OS/version are you using, ...
+- **Tag the Issue with Labels** to categorize it (is it a `bug` or just an `enhancement` like a new feature?). If you're not sure which label to use, leave them empty, someone else will do that (better no labels at all than wrong ones)
+  - `bug` - for broken functionality/UI, bad behaviour
+  - `enhancement/feature` - adding new functionality, tweaks for existing features
+
+It's usually up to Project Manager or someone from the Team to:
+
+- Assign workers to Issues
+- Assign Milestones (AKA "releases") for concrete Issues – please don't assign Milestones to your own Issues if you're not absolutely certain, that the feature/bug _must be_ included in certain release!
+
+---
+
+### 2. Push Requests (PRs)
+
+PRs are working "packages" of code, solving one bug / new feature and are usually connected to existing Issues. For creating new PRs, you must first fork this repository to your own account.
+
+#### 2.1. Create new branch
+
+After your repo is ready, create new branch.
+
+- **Again, keep branch names short and descriptive**
+- Ideally also tag your branch with `prefix/..` – this helps categorizing all the branches at first glance. Some examples:
+  - `fix/..` - for bug fixing, e.g. `fix/createwallet`
+  - `feature/..` - for new features, e.g. `feature/coldstaking`
+  - `gfx/..` - for graphic/UI/UX tweaks, e.g. `gfx/typography`
+
+When your branch is ready, do your changes and push them to the branch. Be sure to push only the changes related to the connected Issue! Solving multiple Issues is also possible, but it's better to focus on one thing at a time.
+
+#### 2.2. Commit your code
+
+- Write meaninful commit messages – don't forget that when your PR is merged, its context is lost. Meaning that once your commits get to `dev` branch, it's harder to say what each commit does just by its name:
```

Comment:
we should mention that the code and commits will be human read with attention and PRs will be refused if they are not described correctly